mora/service/employee.py

```diff
@@ -1010,6 +1010,14 @@ def create_employee():
     except ValueError:
         valid_from = util.negative_infinity
 
+    bruger = c.bruger.fetch(
+        tilknyttedepersoner="urn:dk:cpr:person:{}".format(cpr))
```

Comment:
Bør denne ikke også filtrere på `tilhoerer`? I princippet er der vel intet til hinder for at en bruger importeres til to organisationer?

Reply:
Det er en god pointe - det kigger jeg lige på.